src/Blt/Plugin/Commands/SwsCommands.php

```diff
@@ -465,6 +473,10 @@ public function updateSites($sites, $options = ['partial-config-import' => FALSE
     $sites = explode(',', $sites);
     foreach ($sites as $site_name) {
       $this->switchSiteContext($site_name);
+      if (!$this->isDrupalInstalled()) {
```

Comment:
My only concern with this is a false positive would get a little annoying if it happened in a production release on a bunch of sites. Maybe we could add an override option where it doesn't check if Drupal is installed? That way in the event it does issue false positives we could still trigger the postCodeDeployUpdate manually.

Reply:
Or could we add to the report/slack notifcation the sites that were skipped like we do with failed sites?

Reply:
> Or could we add to the report/slack notifcation the sites that were skipped like we do with failed sites?

I think this would be handy to differentiate between failures and skips.

Reply:
I added an option `require-drupal-install` to the command.